crates/storage/src/blueprint/merklized.rs

```diff
@@ -434,7 +434,7 @@ macro_rules! basic_merklelized_storage_tests {
                 let mut init_storage = InMemoryStorage::default();
                 let mut init_structured_storage = init_storage.write_transaction();
 
-                let mut rng = &mut StdRng::seed_from_u64(1234);
+                let mut rng = &mut StdRng::seed_from_u64(31337);
```

Comment:
this change is needed because with `1234` as the seed, it generates conflicting keys :/

Reply:
That's interesting. Well I guess the three-byte keys result in a pretty high collision risk. Still fun that we've encountered it :sweat_smile: 

This makes me wonder if there's any attack vector where a block producer overrides certain compression keys. Spontaneously it doesn't sound like you would be able to do anything dangerous in that case but it could be worth thinking about.